src/article/models.py

```diff
@@ -26,4 +26,9 @@ class Article(Base):
     )
     author = relationship("Author", back_populates="article")
 
-    comment = relationship("Comment", uselist=False, backref="article")
+    comment = relationship(
+        "Comment",
+        back_populates="article",
+        uselist=False,
+        cascade="all, delete-orphan",
+    )
```

Comment:
you don't need cascade here

why do you use `uselist=False`? Do you know what this thing does?

Each article may have many comments. Not only one !